library/core/src/intrinsics.rs

```diff
@@ -2392,7 +2392,9 @@ extern "rust-intrinsic" {
     pub fn variant_count<T>() -> usize;
 
     /// Rust's "try catch" construct for unwinding. Invokes the function pointer `try_fn` with the
-    /// data pointer `data`, and calls `catch_fn` if unwinding occurs while `try_fn` runs.
+    /// data pointer `data`, and calls `catch_fn` if a panic occurs while `try_fn` runs. (It is
+    /// undefined behavior to catch an unwinding operation from another language with this
+    /// function.)
```

Comment:
```suggestion
    /// data pointer `data`, and calls `catch_fn` if a panic occurs while `try_fn` runs. It is
    /// undefined behavior to catch an unwinding operation from another language with this
    /// function.
```

Important enough that I don't think parens are needed here.